README.adoc

```diff
@@ -1,162 +1,105 @@
 :toc: macro
 
-= Threshold network subgraphs
+= Threshold Network subgraph
 
-This repository contains Threshold network subgraphs.
+This repository contains Threshold Network subgraph.
 
-The subgraphs are open APIs to query data from networks like Ethereum and IPFS. The data is indexed by https://thegraph.com[*The Graph*] decentralized protocol.
+Subgraphs are open APIs to query data from networks like Ethereum and IPFS. The
+data is indexed by https://thegraph.com[*The Graph*] decentralized protocol.
 
 toc::[]
 
+NOTE: Development query URL:
+https://api.studio.thegraph.com/query/24143/threshold/0.0.5
+
+
+== GraphQL relevant entities
+
+- `Account`: represents the base user data: user's stakes and delegations.
+
+- `StakeData`: information (owner, delegates, token amount...) about each stake.
+
+- `Epoch`: each staking event (i.e staked, toppedup and unstaked) creates a new
+epoch, so the duration of these is variable. It contains a list of the stakes at
+that time (EpochStakes).
+
+- `EpochStake`: information about the token amounts staked in each epoch.
+
+- `MinStakeAmount`: minimum amount of tokens allowed to stake in Threshold
+Network.
+
+- `PREOperator`: PRE application stake operator's address and timestamp of
+operator confirmation. If timestamp is null, the operator has been bonded but
+not confirmed.
+
+- `DAOMetric`: metrics about the amount of liquid and staked tokens in DAO.
+
```

Comment:
Non-blocking just a thought: I think we can move the entity docs to a `schema.graphql` file and the docs will be available in the graph studio, for example:
- https://github.com/livepeer/subgraph/blob/main/schema.graphql#L9
- https://thegraph.com/explorersubgraph?id=FDD65maya4xVfPnCjSgDRBz6UBWKAcmGtgY6BmUueJCg&view=Playground

<img width="1260" alt="obraz" src="https://user-images.githubusercontent.com/57687279/178240864-48ea1643-4618-4609-810d-679bfd8f3ca5.png">

Reply:
Good idea! I've just created an issue for this:

https://github.com/threshold-network/subgraph/issues/25